lib/apis/v3/felixconfig.go

```diff
@@ -198,7 +198,11 @@ type FelixConfigurationSpec struct {
 
 	// NATPortRange specifies the range of ports that is used for port mapping when doing outgoing NAT. When unset the default behavior of the
 	// network stack is used.
-	NATPortRange *numorstring.Port `json: natPortRange,omitempty`
+	NATPortRange *numorstring.Port `json:"natPortRange,omitempty"`
+
+	// ExternalNodesCidrList is a list of CIDR's of external-non_calico-nodes which may source tunnel traffic and have
```

Comment:
`non_calico` -> `non-calico` ?

Reply:
Think this should be `ExternalNodesCIDRList` to match Go's normal naming scheme.